src/transformers/tokenization_utils_base.py

```diff
@@ -3299,7 +3299,8 @@ def _pad(
             max_length = ((max_length // pad_to_multiple_of) + 1) * pad_to_multiple_of
 
         needs_to_be_padded = padding_strategy != PaddingStrategy.DO_NOT_PAD and len(required_input) != max_length
-
+        special_token_offset = (0, 0)
```

Comment:
A small nit, but I would give it another name to designate that it is the offset mapping of the pad token. 

How about `pad_token_offset`?

Reply:
Yeah, good variant! Do I need something to do yet, e.g tests?

Reply:
> Do I need something to do yet, e.g tests?

Yes indeed these changes will have to be included in this PR :relaxed:

Reply:
Ok